Emby.Server.Implementations/Security/AuthenticationRepository.cs

```diff
@@ -362,31 +351,28 @@ private static AuthenticationInfo Get(IReadOnlyList<IResultSetValue> reader)
 
         public DeviceOptions GetDeviceOptions(string deviceId)
         {
-            using (WriteLock.Read())
+            using (var connection = GetConnection(true))
             {
-                using (var connection = CreateConnection(true))
+                return connection.RunInTransaction(db =>
```

Comment:
Given this is a read only query (SELECT), you might have to remove the RunInTransaction to avoid errors like 'cannot start transaction in transaction'.

Reply:
I'm keeping my hands of these for now.